src/auth/user-import-builder.ts

```diff
@@ -359,7 +359,7 @@ export class UserImportBuilder {
         rounds,
       };
       break;
-
+    }
     case 'PBKDF_SHA1':
     case 'PBKDF2_SHA256':
```

Comment:
Why are the braces `{ }` used only for some `case:` clauses and not others?

Reply:
It's only used for case blocks that declare variables with const or let.

See https://eslint.org/docs/rules/no-case-declarations